experiments/ClimaEarth/test/debug_plots_tests.jl

```diff
@@ -35,7 +35,7 @@ plot_field_names(sim::Interfacer.SurfaceStub) = (:stub_field,)
 @testset "import_atmos_fields!" begin
 
     boundary_space = TestHelper.create_space(FT)
-    coupler_names = [
+    coupler_names = (
```

Comment:
Just curious, why did you change this from a vector to a tuple?

Reply:
Oops, it was a mistake!